src/go/k8s/cmd/main.go

```diff
@@ -351,11 +351,11 @@ func main() {
 			helmRelease := helmController.HelmReleaseReconcilerFactory{
 				Client:           mgr.GetClient(),
 				EventRecorder:    helmReleaseEventRecorder,
-				Metrics:          metricsH,
-				GetClusterConfig: ctrl.GetConfig,
-				FieldManager:     controllerName,
 				ClientOpts:       clientOptions,
 				KubeConfigOpts:   kubeConfigOpts,
+				FieldManager:     "redpanda-helmRelease-controller",
```

Comment:
Can we make a const at the top of this file instead of using magic strings? Is there any reason we're mixing kabob casing and camel casing here? That seems very likely to cause errors and typos later on.

Reply:
no reason really, but sure we can move them to vars.